src/js/components/Meter/Bar.js

```diff
@@ -60,15 +62,49 @@ const Bar = forwardRef((props, ref) => {
           start -= delta;
         }
 
+        let stroke = strokeProps(
+          someHighlight && !highlight ? background : colorName,
+          theme,
+        );
+
+        const useGradient = color && Array.isArray(color);
+
+        if (useGradient) {
```

Comment:
```suggestion
        if (color && Array.isArray(color)) {
```
Since we only use this variable one I think we don't have to explicitly define it